src/tests/test_jitter_rng.cpp

```diff
@@ -0,0 +1,37 @@
+/*
+* (C) 2024 Planck Security S.A.
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#include <vector>
+
+#include <botan/build.h>
+
+#ifdef BOTAN_HAS_JITTER_RNG
+
+#include <botan/jitter_rng.h>
+
+#include "tests.h"
+
+namespace Botan_Tests
+{
+class Jitter_RNG_Tests final : public Test {
+
+    std::vector<Test::Result> run() override {
+        int sample_count = 512;
```

Comment:
As a (slightly) improved test, check that `Jitter_RNG` can output strings within any range 0..512 ie

```
for(size_t sample_count = 0; sample_count <= 512; ++sample_count) {
   …
}
```